lib/vault/errors.rb

```diff
@@ -18,6 +18,19 @@ def initialize
     $ export VAULT_TOKEN="..."
 
 Please refer to the documentation for more examples.
+EOH
+    end
+  end
+
+  class MissingRequiredStateError < VaultError
+    def initialize
+      super <<-EOH
+The performance standby node does not yet have the 
+most recent index state required to authenticate 
+the request.
+
+The request should be retried with the with_retries clause.
```

Comment:
* I've been quickly skimming the behavior of `RETRIED_EXCEPTIONS` and I _think_ that exception list is only retried with `with_retries`, is that right? An `HTTPServerError` is _not_ retried without the `with_retries` block, currently?

* If the above is true, I can see giving the actionable advice here at the end. Is this always true, though? Any reason a client may be wanting to see an explicit error returned here? Maybe `Generally, the request should be retried...`?

Reply:
RETRIED_EXCEPTIONS seems to just be the default value used _if_ no error types were passed as parameters to `with_retries`. So if for some reason someone wanted to see the explicit error, they could either make a request without `with_retries`, or call `with_retries` with a list of params to retry on that doesn't include `MissingRequiredStateError`.

I agree though about the messaging, I'll change it to "Generally" like you said.